ADAL/src/utils/ios/ADKeychainUtil.m

```diff
@@ -53,15 +53,16 @@ + (NSString*)retrieveTeamIDFromKeychain:(ADAuthenticationError * __autoreleasing
     NSDictionary *query = @{ (id)kSecClass : (id)kSecClassGenericPassword,
                              (id)kSecAttrAccount : @"teamIDHint",
```

Comment:
We renamed this item. In older SDKs it was "bundleSeedID" check your code to see what else might be adding "teamIDHint"